lib/feeder_importer.rb

```diff
@@ -139,29 +143,50 @@ def create_series
       num_segments = [episode.media_resources.count, num_segments].max
     end
 
-    self.template = series.audio_version_templates.create!(
+    contains_video = podcast.episodes.any?(&:contains_video?)
+
+    self.audio_template = series.audio_version_templates.create!(
       label: "Podcast Audio #{num_segments} #{'segment'.pluralize(num_segments)}",
+      content_type: AudioFile::MP3_CONTENT_TYPE,
       segment_count: num_segments,
       promos: false,
       length_minimum: 0,
       length_maximum: 0
     )
 
-    num_segments.times do |x|
-      num = x + 1
-      template.audio_file_templates.create!(
-        position: num,
-        label: "Segment #{num}",
+    if contains_video
+      self.video_template = series.audio_version_templates.create!(
+        label: "Podcast Video #{num_segments} #{'segment'.pluralize(num_segments)}",
+        content_type: AudioFile::VIDEO_CONTENT_TYPE,
+        segment_count: num_segments,
+        promos: false,
         length_minimum: 0,
         length_maximum: 0
       )
     end
 
+    templates = [video_template, audio_template].compact
+
+    templates.each do |t|
+      num_segments.times do |x|
+        num = x + 1
+        t.audio_file_templates.create!(
+          position: num,
+          label: "Segment #{num}",
+          length_minimum: 0,
+          length_maximum: 0
+        )
+      end
+    end
+
     self.distribution = Distributions::PodcastDistribution.create!(
       url: "#{feeder_root}/podcasts/#{podcast_id}",
       distributable: series,
-      audio_version_template: template
+      audio_version_template: audio_template
```

Comment:
you can set both templates in there now - it used to support only one, but a distro can now have multiple templates.